### ISSUE TITLE
ES Modules

### ISSUE THREAD

**Issue by salzig** [eshepelyuk/semantic-release-helm-oci#4]
Hej, 
using this plugin you'll nowadays get an Error like: 
```
Error [ERR_REQUIRE_ESM]: require() of ES Module /usr/lib/node_modules/@semantic-release/error/index.js from /usr/lib/node_modules/@eshepelyuk/semantic-release-helm-oci/lib/verifyConditions.js not supported.
Instead change the require of index.js in /usr/lib/node_modules/@eshepelyuk/semantic-release-helm-oci/lib/verifyConditions.js to a dynamic import() which is available in all CommonJS modules.
```

which is related to how require/import/includes are handled in the JavaScript world. 

Do you have time to work on the necessary change? Are you interested in a MergeRequest? 

related Semantic-Release Issue: https://github.com/semantic-release/semantic-release/issues/2133

**Comment by eshepelyuk** [comment]
Hello @salzig 

Thanks for reporting.
If you are able to provide a PR - it's welcome.

**Comment by eshepelyuk** [comment]
Hello @salzig 
Should be available in 2.0.0.

Please give it a try and close the issue if everything is OK.

**Comment by salzig** [comment]
```
[12:36:57 PM] [semantic-release] › ℹ  Start step "prepare" of plugin "@eshepelyuk/semantic-release-helm-oci"
[12:36:57 PM] [semantic-release] [@eshepelyuk/semantic-release-helm-oci] › ℹ  Chart packaged to webapp-0.1.0.tgz.
[12:36:57 PM] [semantic-release] › ✔  Completed step "prepare" of plugin "@eshepelyuk/semantic-release-helm-oci"
[12:36:58 PM] [semantic-release] › ✔  Created tag v0.1.0
[12:36:58 PM] [semantic-release] › ℹ  Start step "publish" of plugin "@eshepelyuk/semantic-release-helm-oci"
[12:36:59 PM] [semantic-release] [@eshepelyuk/semantic-release-helm-oci] › ℹ  Chart published to oci://registry.exmaple.com/maintenance/charts.
[12:36:59 PM] [semantic-release] › ✔  Completed step "publish" of plugin "@eshepelyuk/semantic-release-helm-oci"
[12:36:59 PM] [semantic-release] › ✔  Published release 0.1.0 on default channel
```
 
 looks fine
